packages/components/menu/src/MenuList/MenuList.tsx

```diff
@@ -8,7 +8,7 @@ import { MenuListHeader } from './MenuListHeader';
 import { MenuListFooter } from './MenuListFooter';
 
 interface MenuListInternalProps extends CommonProps {
-  children?: React.ReactNodeArray;
+  children?: React.ReactNodeArray | React.ReactNode;
```

Comment:
```suggestion
  children?: React.ReactNode;
```
In such case we probably don't need `React.ReactNodeArray` at all